storey/drivers.py

```diff
@@ -501,3 +509,450 @@ async def _get_item(self, container, table_path, key, attributes):
+                                    lua_script = f'{lua_script}len=redis.call("LLEN",list_attribute_key) for i=1,({bucket.total_number_of_buckets}-len) \
+                                        do redis.call("RPUSH",list_attribute_key,{default_value}) end;\n'
+                                if array_time_attribute_key not in times_updates:
+                                    times_updates[array_time_attribute_key] = expected_time_expr
+                                new_cached_times[name] = (array_time_attribute_key, expected_time)
+
+                            # Updating the specific cells
+                            if cached_time <= expected_time:
+                                # condition = None
+                                # if partitioned_by_key and self._mtime_name in aggregation_element.storage_specific_cache:
+                                #     condition = aggregation_element.storage_specific_cache[self._mtime_name]
+                                lua_script = f'{lua_script}local old_value=redis.call("LINDEX", list_attribute_key, {index_to_update});\
+                                    old_value=tonum(old_value);\n'
+                                lua_script = f'{lua_script}old_value=tonum(old_value)\n'
+                                new_value_expression = aggregation_value.aggregate_lua_script('old_value', aggregation_value.value)
+                                lua_script = f'{lua_script}\
+                                    redis.call("LSET", list_attribute_key, {index_to_update}, {new_value_expression});\n'
+
+                        redis_keys_involved.append(array_time_attribute_key)
+                        lua_script = f'{lua_script}redis.call("SET","{array_time_attribute_key}",{expected_time}); \n'
+
+        return lua_script, condition_expression, pending_updates, redis_keys_involved
+
+    async def _save_key(self, container, table_path, key, aggr_item, partitioned_by_key, additional_data):
+        redis_key_prefix = self.make_key(container, table_path, key)
+        update_expression, mtime_condition, pending_updates, redis_keys_involved = self._build_feature_store_lua_update_script(
+            redis_key_prefix, aggr_item, partitioned_by_key, additional_data)
+        if not update_expression:
+            return
+        current_time = int(time.time_ns()/1000)
+        if mtime_condition is not None:
+            update_expression = f'if redis.call("HGET", "{redis_key_prefix}","{self._mtime_name}") == "{mtime_condition}" then \
+                {update_expression} redis.call("HSET","{redis_key_prefix}","{self._mtime_name}",{current_time});return 1;else return 0;end;'
+        else:
+            update_expression = f'{update_expression}redis.call("HSET","{redis_key_prefix}","{self._mtime_name}",{current_time});return 1;'
+
+        redis_keys_involved.append(redis_key_prefix)
+        update_expression = f"""{update_expression}"""
```

Comment:
This is really just
```suggestion
        update_expression = str(update_expression)
```

Reply:
Is it? I need the " to be part of the result string